extensions/transform/org.eclipse.smarthome.transform.xpath/README.md

```diff
@@ -1,3 +1,52 @@
 # XPath Transformation Service
 
-Transform XML input using an [XPath expression](https://www.w3.org/TR/xpath/#section-Expressions).
+Transforms an [XML](https://www.w3.org/XML/) input using an [XPath](https://www.w3.org/TR/xpath/#section-Expressions) expression.
+
+## Advanced Example
+Given a retrived XML e.g. from an HIK Vision device with the namespace `xmlns="http://www.hikvision.com/ver20/XMLSchema"`.
```

Comment:
Given a retrieved XML (e.g. from an HIK Vision device with the namespace `xmlns="http://www.hikvision.com/ver20/XMLSchema"`.):